docs/src/data/frameworks.ts

```diff
@@ -27,9 +27,9 @@ export const REACT_NATIVE_DEPENDENCIES =
   'react-native-safe-area-context amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill';
 
 export const FRAMEWORK_INSTALL_SCRIPTS = {
-  react: 'npm i @aws-amplify/ui-react aws-amplify',
-  vue: 'npm i @aws-amplify/ui-vue aws-amplify',
-  angular: 'npm i @aws-amplify/ui-angular aws-amplify',
+  react: '@aws-amplify/ui-react aws-amplify',
+  vue: '@aws-amplify/ui-vue aws-amplify',
+  angular: '@aws-amplify/ui-angular aws-amplify',
   flutter: 'flutter pub add amplify_authenticator',
-  'react-native': `npm i @aws-amplify/ui-react-native aws-amplify ${REACT_NATIVE_DEPENDENCIES}`,
+  'react-native': `@aws-amplify/ui-react-native aws-amplify ${REACT_NATIVE_DEPENDENCIES}`,
```

Comment:
Can we make flutter more consistent with the others?

Reply:
I don't think so because flutter dependencies can't be installed using npm or yarn as others https://pub.dev/packages/amplify_authenticator/install

Reply:
Yes -- I just found it weird that the others just list the dependencies, whereas flutter one has the install commands prepended.

Reply:
So removing the `flutter pub add` and put it in TerminalCommand component like how we handle the others right?

Reply:
I see...

Reply:
Done!